homeassistant/components/switcher_kis/config_flow.py

```diff
@@ -27,6 +43,9 @@ async def async_step_user(
         if self._async_current_entries(True):
             return self.async_abort(reason="single_instance_allowed")
 
+        if user_input is None:
+            return await self._show_setup_form(user_input)
```

Comment:
Isn't Param 2 for this function errors?
Also no need to pass it if you know it is None.

Reply:
changed stuff. check again.